examples/digitalidentity/app/Http/Controllers/ReceiptController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Yoti\DigitalIdentityClient;
+use Yoti\YotiClient;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller as BaseController;
+use Yoti\Profile\Attribute;
+use Yoti\Profile\UserProfile;
+use Yoti\Util\Logger;
+class ReceiptController extends BaseController
+{
+    public function show(Request $request, DigitalIdentityClient $client, ?LoggerInterface $logger = null)
+    {
+        $logger = $logger ?? new Logger();
+
+        $logger->warning("Unknown Content Type parsing as a String");
+        $activityDetails = $client->fetchShareReceipt($request->query('ReceiptID'));
```

Comment:
It would make sense to rename `$activityDetails` to  `$shareReceipt`.

Reply:
renamed